command/quota_init.go

```diff
@@ -106,30 +106,22 @@ func (c *QuotaInitCommand) Run(args []string) int {
 
 var defaultHclQuotaSpec = strings.TrimSpace(`
 name = "default-quota"
-description = "Limit the shared default namespace"
+description = "Unlimited default"
 
-# Create a limit for the global region. Additional limits may
-# be specified in-order to limit other regions.
 limit {
     region = "global"
-    region_limit {
-        cpu = 2500
-        memory = 1000
-    }
+    region_limit {}
```

Comment:
I agree this is the right value to have, but for purposes of using `nomad quota init` to get started, maybe we can comment-out the other value as an example? Similar to what we do with `nomad job init`